src/main/scala/com/decodified/scalassh/Command.scala

```diff
@@ -18,11 +18,26 @@ package com.decodified.scalassh
 
 import java.io.{ByteArrayInputStream, File, FileInputStream, InputStream}
 import net.schmizz.sshj.connection.channel.direct.Session
-
-final case class Command(command: String, input: CommandInput = CommandInput.NoInput, timeout: Option[Int] = None)
+trait Command {
```

Comment:
Let's make this a `sealed` trait.

Reply:
Overall this was reverted, perhaps i should be a different PR.